### ISSUE TITLE
Missing 'features' slot filling in 'similar_celltypes.chromatinAccessibility' 

### ISSUE THREAD

**Issue by YingX97** [fabilab/cell_atlas_approximations_NLP#31]
Hi Fabio,

Q: "what are cell types in chromatin accessibility similar to lung fibroblast in human".

It doesn't gives proper follow up question, so I couldn't specific what genes I want to use.

**Comment by YingX97** [comment]
Hey Fabio,

The follow up question is still missing. 
I am using the lastest NLP (v0.2.27). In the response returned by the `nlp.ask` function, the NLP recognized both "organ" and "cell type" as "lung.

**Comment by YingX97** [comment]
Testing this issue on: 
Q: "what are cell types in chromatin accessibility similar to lung fibroblast in human".

follow up question: 
"You requested similar cell types: how many?" --- 2

"Please list the features (e.g. genes) you would like to look into?" --- chr1:9955-10355, chr10:122199710-122200110

**Comment by YingX97** [comment]
fixed in @fabilab/atlasapprox-nlp 3.6
